packages/material-ui/src/utils/unstable_useId.js

```diff
@@ -1,18 +1,22 @@
 import * as React from 'react';
 
+/**
+ * @ignore - internal component.
```

Comment:
Some component should be public to configure the id generator. Otherwise this feature wouldn't be useful.